job/common/src/main/java/alluxio/job/workflow/composite/CompositeConfig.java

```diff
@@ -29,7 +30,7 @@
  * execute a set of jobs concurrently, each job can run run another set of sub jobs sequentially.
  */
 @ThreadSafe
-public final class CompositeConfig implements JobConfig {
+public final class CompositeConfig implements PlanConfig {
```

Comment:
Will everything be a composite config? Does it make sense to have a single `workflow` package instead of `workflow.composite`?

Reply:
If all dependencies will be based around a predefined set of jobs, then it would be the case. However, I don't expect that to be the case for long. For example, I plan to rewrite MigrateDefinition's parallelism to be defined by some file system iteration config with specified depth. Which would be a workflow but with a very different implementation.

Reply:
Should `CompositeConfig` implement `WorkloadConfig`?